sdk/servicebus/service-bus/test/stress/Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM node:8-alpine
```

Comment:
We can definitely bump this up now that node 8 is not our platform minimum. @chradek , should we just skip straight to node 12?